zero/client_server/client.py

```diff
@@ -231,7 +231,10 @@ async def _poll_data():
 
         resp_data = self._resp_map.pop(req_id)
 
-        if isinstance(resp_data, dict) and "__zerror__method_not_found" in resp_data:
-            raise MethodNotFoundException(resp_data.get("__zerror__method_not_found"))
+        if isinstance(resp_data, dict):
+            if e := resp_data.get("__zerror__method_not_found"):
+                raise MethodNotFoundException(e)
+            if e := resp_data.get("__zerror__exception"):
```

Comment:
Maybe name it something like `__zerror__server_exception` or `__zerror__remote_exception`? 🤔